shared/date/date.properties.po

```diff
@@ -1,246 +1,244 @@
-#. # Day names
-#. extracted from ./shared/date/date.properties
 msgid ""
 msgstr ""
-"Project-Id-Version: PACKAGE VERSION\n"
+"Project-Id-Version: \n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2015-02-11 19:45+0000\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
```

Comment:
Poedit > Catalog> Properties

The standard i'd like us to use is:
Project name FirefoxOS
Team: Ndebele
Team email: contact@mozilla-zimbabwe.org
Language: nd
Charset UTF-8

Please also add the above standard to our documentation, i.e the.md file